### ISSUE TITLE
[NVDA - Stream - Inaccessible Video]: Mute button in the video is not accessible using keyboard when NVDA is enabled.

### ISSUE THREAD

**Issue by HarishTerala** [nvaccess/nvda#9404]
### Steps to reproduce: 
Step 1: Start NVDA.
Step 2: Open the URL( https://web-srol-1.stream.azure-test.net/) using Firefox browser and login with valid credentials
Step 3: Now "Stream" home page opens.
Step 4: Navigate to any video and press ENTER to play.
Step 5: Now navigate to Mute video control.
Step 6: Observe whether Mute video control is able to access using Keyboard when NVDA is enabled.

### Actual behavior:
Mute Video control is not able to access using keyboard when NVDA is enabled. when clicking ENTER on mute video control, video is getting "Pause".  

### Expected behavior:
Mute Video control should able to access using keyboard when NVDA is enabled. when clicking ENTER on any of the control, respected control should be activated.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2018.4.1

#### Windows version:
1803 (OS Build 17728.1000)

#### Name and version of other software in use when reproducing the issue:
Firefox version (60.0)
#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by fisher729** [comment]
Hi.

Which website is that specifically? I'm not seeing any trace of it on my end.

**Comment by Brian1Gaff** [comment]
No not here either.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by DrSooom** [comment]
@HarishTerala: Please add the mentioned URL and the Firefox version to the issue description. Thanks.

**Comment by HarishTerala** [comment]
Updated the bug by providing URL and Firefox version .Thanks.

**Comment by Adriani90** [comment]
@HarishTerala the link requires an organisational account. This is not possible to reproduce for everyone. Could you please post a link where everyone can reproduce this issue?
Also please give us an update if this issue occurs in NVDA 2020.1. Thanks.

**Comment by ehollig** [comment]
@HarishTerala we are not able to access the link as it requires a login. If you wish, you can provide the login information privately to NvAccess in an email to investigate. I am closing this issue, unless a publicly available URL of the video player can be provided.
